sdk/security_keyvault/src/clients/certificate_client.rs

```diff
@@ -250,6 +279,38 @@ impl CertificateClient {
         DeleteCertificateBuilder::new(self.clone(), name.into())
     }
 
+    /// Deletes a certificate in the Key Vault.
+    ///
+    /// # Arguments
+    ///
+    /// * `name` - Name of the certificate
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// use azure_security_keyvault::KeyvaultClient;
+    /// use azure_identity::DefaultAzureCredential;
+    /// use tokio::runtime::Runtime;
+    /// use std::sync::Arc;
+    ///
+    /// async fn example() {
+    ///     let creds = DefaultAzureCredential::default();
+    ///     let mut client = KeyvaultClient::new(
+    ///         &"KEYVAULT_URL",
+    ///         Arc::new(creds),
+    ///     ).unwrap().certificate_client();
+    ///     client.delete_operation("NAME").await.unwrap();
+    /// }
+    ///
+    /// Runtime::new().unwrap().block_on(example());
+    /// ```
+    pub fn delete_operation<N>(&self, name: N) -> DeleteCertificateOperationBuilder
```

Comment:
To somewhat align with our other languages, this should be `delete_certificate`. In other lanugages we actually define this as a long-running operation (LRO), but only because DELETE returns immediately but may not take effect immediate. It's a pseudo-LRO and I think, at this point, it's not worth replicating here.

Reply:
Sorry for the confusion. I forgot to update the documentation above this function. This function deletes the creation operation for a specific certificate not the certificate itself. https://learn.microsoft.com/en-us/rest/api/keyvault/certificates/delete-certificate-operation/delete-certificate-operation?tabs=HTTP

Reply:
I see. Eventually these might have to change to better align with our other languages' SDKs, but this works for now if that unblocks you.